### ISSUE TITLE
Cli command arg 'xlabel' & 'ylabel' seems to be reversed.

### ISSUE THREAD

**Issue by ysjjovo** [mcastorina/graph-cli#54]
this is my shell command.
```
graph result.csv --xlabel='threads' --ylabel='q/s' -o write.png
```
'xlabel' was displayed at 'ylabel' position in the generated image 'write.png'

**Comment by mcastorina** [comment]
Thanks for catching this. Looks like it's a bug introduced in f262baaf95a03f4dc4ec36d3badce1ff02bfe85e

Should be an easy fix :)

**Comment by mcastorina** [comment]
Fixed in 5f1a0c3c2e24f84cf84b015a2a9dae7d3cd86e66 and I pushed v0.1.18 with the change.

Thanks again!
